src/containers/Search.tsx

```diff
@@ -0,0 +1,39 @@
+import { Search as SearchIcon } from "react-bootstrap-icons";
+import Form from "react-bootstrap/Form";
+import InputGroup from "react-bootstrap/InputGroup";
```

Comment:
Why  you dont use destructuring like "import {Form,  InputGroup} from "react-bootstrap";"?
one line, less code.

Reply:
Import in VS code works as default like this.